modules/apps/portal-workflow/portal-workflow-web/src/main/resources/META-INF/resources/js/main.js

```diff
@@ -163,13 +163,11 @@ AUI.add(
 				for (var index in elementsList) {
 					var element = elementsList[parseInt(index, 10)];
 
-					var hidden = element.getAttribute('hidden');
-
-					if (hidden) {
-						element.removeAttribute('hidden');
+					if (element.classList.contains('d-none')) {
+						element.classList.remove('d-none');
 					}
 					else {
-						element.setAttribute('hidden', true);
+						element.classList.add('d-none');
 					}
```

Comment:
I also tried with
```js
if (element.hasClass('d-none')) {
    element.removeClass('d-none');
}
else {
    element.addClass('d-none');
}
```

after reading these docs
- https://alloyui.com/api/classes/A.Node.html#method_removeClass
- https://alloyui.com/api/files/yui3_src_node_js_node-class.js.html#l24

but it didn't work

Reply:
There's better ways to do this in AUI, but here you don't have `Y.Node` but regular `HTMLElements` coming from down in a `document.getElementById(arguments[index]);`, so those methods are not available there.

I think this should be fine for now as it is.